storage: correct struct fields in volume plugins

humblec opened the pull request:
This commit corrects struct field names in the godoc for various
storage volume plugins volume sources and persistent volume sources.

Additional Ref# #105963 (comment)

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>


/kind cleanup

```release-note
NONE

humblec committed:
storage: correct struct fields in volume plugins

This commit corrects struct field names in the godoc for various
storage volume plugins volume sources and persistent volume sources.

Additional Ref# #105963 (comment)

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented:
/sig storage

humblec commented:
/release-note-none
/assign @liggitt

liggitt commented on staging/src/k8s.io/api/core/v1/types.go:
```diff
@@ -718,15 +718,15 @@ type EmptyDirVolumeSource struct {
 // Glusterfs volumes do not support ownership management or SELinux relabeling.
 type GlusterfsVolumeSource struct {
-	// EndpointsName is the endpoint name that details Glusterfs topology.
+	// endpointsName is the endpoint name that details Glusterfs topology.
```
```suggestion
	// endpoints is the endpoint name that details Glusterfs topology.
```

liggitt commented on staging/src/k8s.io/api/core/v1/types.go:
```diff
@@ -1247,42 +1247,42 @@ type NFSVolumeSource struct {
 	// iSCSI Interface Name that uses an iSCSI transport.
 	// Defaults to 'default' (tcp).
 	// +optional
 	ISCSIInterface string `json:"iscsiInterface,omitempty" protobuf:"bytes,4,opt,name=iscsiInterface"`
```
update doc to `iscsiInterface ...`

liggitt commented on staging/src/k8s.io/api/core/v1/types.go:
```diff
@@ -1339,78 +1339,78 @@ type ISCSIPersistentVolumeSource struct {
+	// secretName is the  name of secret that contains Azure Storage Account Name and Key
 	SecretName string `json:"secretName" protobuf:"bytes,1,opt,name=secretName"`
-	// Share Name
+	// sharename is the azure share Name
```
```suggestion
	// shareName is the azure share Name
```

liggitt commented on staging/src/k8s.io/api/core/v1/types.go:
```diff
@@ -1339,78 +1339,78 @@ type ISCSIPersistentVolumeSource struct {
+	// sharename is the azure share Name
 	ShareName string `json:"shareName" protobuf:"bytes,2,opt,name=shareName"`
-	// Defaults to false (read/write). ReadOnly here will force
+	// readONly defaults to false (read/write). ReadOnly here will force
```
```suggestion
	// readOnly defaults to false (read/write). ReadOnly here will force
```

liggitt commented on staging/src/k8s.io/api/core/v1/types.go:
```diff
@@ -1606,7 +1606,7 @@ type StorageOSPersistentVolumeSource struct {
 type ConfigMapVolumeSource struct {
 	LocalObjectReference `json:",inline" protobuf:"bytes,1,opt,name=localObjectReference"`
-	// If unspecified, each key-value pair in the Data field of the referenced
+	// Items if unspecified, each key-value pair in the Data field of the referenced
```
```suggestion
	// items if unspecified, each key-value pair in the Data field of the referenced
```

liggitt commented on staging/src/k8s.io/api/core/v1/types.go:
```diff
@@ -1699,16 +1699,16 @@ type ProjectedVolumeSource struct {
 	// +optional
 	DownwardAPI *DownwardAPIProjection `json:"downwardAPI,omitempty" protobuf:"bytes,2,opt,name=downwardAPI"`
-	// information about the configMap data to project
+	// configmap information about the configMap data to project
```
```suggestion
	// configMap information about the configMap data to project
```

liggitt reviewed (commented):
noted some missing/typoed field names. also do a sweep to make sure what follows the field name is a coherent phrase

humblec commented:
> noted some missing/typoed field names. also do a sweep to make sure what follows the field name is a coherent phrase

Sharp as always :) , sure, addressed those comments .. Thanks @liggitt, ptal !

liggitt commented:
/lgtm
/approve
